m4_sessions/1705-inning/group_six.md

```diff
@@ -10,5 +10,13 @@
 
 ### Outline: 
 
-<p>10 minutes for answering questions. 15 minutes whiteboard process. 25 minutes group discussions. 10 minutes for conclusion. </p>
+  * Answer question on their own (10 minutes):
+     * What is knowledge?
+     
+  * Whiteboard (15 minutes)
+    * Symbolic logic exercise
+  
+  * Group discussions on their answer to question ( 25 minutes )
 
+  * Conclusion ( 10 minutes )
+    * [video](https://www.youtube.com/watch?v=kXhJ3hHK9hQ)
```

Comment:
Why does this matter for students and your future career as a software developer?  I think you are missing that connection.  This is super interesting, but can you draw a stronger understanding or deeper connection to why this matters as developers?